public/stylesheets/style.css

```diff
@@ -0,0 +1,24 @@
+
+* {
```

Comment:
I would like to avoid the use of globally applied styles (unless we decide it necessary in our style guide)